main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -30,6 +32,12 @@ object JvmOps {
   val RootPackage: List[String] = Nil
 
   /**
+    * Version of Jave
+    */
+  val JavaVersion = V1_8
```

Comment:
I will introduce a separate AsmOps and we can move ASM related helpers into that.